<!-- BEGIN KEY -->
public class javax.naming.SizeLimitExceededException extends javax.naming.LimitExceededException
<!-- END KEY -->
<!-- BEGIN DATA -->
当某一方法生成的结果超出与大小有关的限制时，抛出此异常。例如，如果结果中包含的对象多于用户所请求的数量，或者结果的大小超出了某个特定于实现的限制时，可能出现这种情况。
 <p>
应用于 NamingException 的同步和序列化问题在这里可以直接应用。

@author Rosanna Lee
@author Scott Seligman

@version 1.8 05/11/17
@since 1.3

<!-- END DATA -->
<!-- BEGIN KEY -->
public javax.naming.SizeLimitExceededException()
<!-- END KEY -->
<!-- BEGIN DATA -->
构造一个新的 SizeLimitExceededException 实例。所有字段都默认为 null。

<!-- END DATA -->
<!-- BEGIN KEY -->
public javax.naming.SizeLimitExceededException(String explanation)
<!-- END KEY -->
<!-- BEGIN DATA -->
使用解释构造一个新的 SizeLimitExceededException 实例。其他所有字段都默认为 null。

@param explanation 关于此异常的详细信息（可能为 null）。

<!-- END DATA -->
